### PR TITLE
More mappings

### PR DESCRIPTION
**Pull request by pyrox0**
Woohoo we love mapping!!!

### CODE REVIEW

File: mappings/net/minecraft/client/option/GameOptions.mapping [hunk 1]
```diff
@@ -115,6 +118,7 @@ CLASS net/minecraft/unmapped/C_lscptbhn net/minecraft/client/option/GameOptions
 	FIELD f_nrolbpfd narratorHotkey Lnet/minecraft/unmapped/C_vbibxkxv;
 	FIELD f_nrvqsydf client Lnet/minecraft/unmapped/C_ayfeobid;
 	FIELD f_ntefrlbi ALLOW_SERVER_LISTINGS_TOOLTIP Lnet/minecraft/unmapped/C_rdaqiwdt;
+	FIELD f_nvenypzh MENU_BACKGROUND_BLURRINESS_DEFAULT_VALUE D
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/option/GameOptions.mapping]:
```suggestion
	FIELD f_nvenypzh DEFAULT_MENU_BACKGROUND_BLURRINESS D
```

File: mappings/net/minecraft/client/world/WorldCreator.mapping [hunk 2]
```diff
@@ -14,6 +14,7 @@ CLASS net/minecraft/unmapped/C_hypraqfe net/minecraft/client/world/WorldCreator
 	FIELD f_sleyatvc extendedWorldTypes Ljava/util/List;
 	FIELD f_tyljjtrt difficulty Lnet/minecraft/unmapped/C_mpbjgxic;
 	FIELD f_xaojvoii DEFAULT_WORLD_NAME Lnet/minecraft/unmapped/C_rdaqiwdt;
+	FIELD f_ymncqayj areCommandsAllowed Ljava/lang/Boolean;
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/world/WorldCreator.mapping]:
```suggestion
	FIELD f_ymncqayj allowCommands Ljava/lang/Boolean;
```

File: mappings/net/minecraft/client/world/WorldCreator.mapping [hunk 3]
```diff
@@ -54,6 +55,8 @@ CLASS net/minecraft/unmapped/C_hypraqfe net/minecraft/client/world/WorldCreator
 		ARG 2 worldDimensions
 	METHOD m_lxkbugsk isDebug ()Z
 	METHOD m_mswhmzta getGameMode ()Lnet/minecraft/unmapped/C_hypraqfe$C_ohbtvxdv;
+	METHOD m_ombzuznr setAreCommandsAllowed (Z)V
+		ARG 1 areCommandsAllowed
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/client/world/WorldCreator.mapping]:
```suggestion
	METHOD m_ombzuznr setAllowCommands (Z)V
		ARG 1 allow
```

File: mappings/net/minecraft/command/argument/ItemPredicateReader.mapping [hunk 4]
```diff
@@ -0,0 +1,24 @@
+CLASS net/minecraft/unmapped/C_ppnqjqhp net/minecraft/command/argument/ItemPredicateReader
+	FIELD f_dhelxfdc SUGGESTIONS_BUILDER Ljava/util/function/Function;
+	FIELD f_gkzxymrz EXPECTED_COMPONENT_ARGUMENT_ITEM Lcom/mojang/brigadier/exceptions/SimpleCommandExceptionType;
+	FIELD f_hhuqrouq MALFORMED_COMPONENT_ARGUMENT_ITEM Lcom/mojang/brigadier/exceptions/Dynamic2CommandExceptionType;
+	FIELD f_klnzxkqr REPEATED_COMPONENT_ARGUMENT_ITEM Lcom/mojang/brigadier/exceptions/DynamicCommandExceptionType;
+	FIELD f_peywtmdd UNKNOWN_ITEM_ARGUMENT_TAG Lcom/mojang/brigadier/exceptions/DynamicCommandExceptionType;
+	FIELD f_purwidzp UNKNOWN_COMPONENT_ARGUMENT_ITEM Lcom/mojang/brigadier/exceptions/DynamicCommandExceptionType;
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/command/argument/ItemPredicateReader.mapping]:
```suggestion
	FIELD f_gkzxymrz ITEM_COMPONENT_EXPECTED Lcom/mojang/brigadier/exceptions/SimpleCommandExceptionType;
	FIELD f_hhuqrouq ITEM_COMPONENT_MALFORMED Lcom/mojang/brigadier/exceptions/Dynamic2CommandExceptionType;
	FIELD f_klnzxkqr ITEM_COMPONENT_REPEATED Lcom/mojang/brigadier/exceptions/DynamicCommandExceptionType;
	FIELD f_peywtmdd ITEM_TAG_UNKNOWN Lcom/mojang/brigadier/exceptions/DynamicCommandExceptionType;
	FIELD f_purwidzp ITEM_COMPONENT_UNKNOWN Lcom/mojang/brigadier/exceptions/DynamicCommandExceptionType;
```

File: mappings/net/minecraft/command/argument/ItemPredicateReader.mapping [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+	FIELD f_qnneocgc item Lnet/minecraft/unmapped/C_vtbxyypo$C_drwjtlvl;
+	FIELD f_wafqtzoh INVALID_ITEM_ARGUMENT_ID Lcom/mojang/brigadier/exceptions/DynamicCommandExceptionType;
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/command/argument/ItemPredicateReader.mapping]:
```suggestion
	FIELD f_wafqtzoh ITEM_ID_INVALID Lcom/mojang/brigadier/exceptions/DynamicCommandExceptionType;
```

File: mappings/net/minecraft/command/argument/ItemSlotArgumentType.mapping [hunk 6]
```diff
@@ -1,6 +1,7 @@
 CLASS net/minecraft/unmapped/C_jqbfufjx net/minecraft/command/argument/ItemSlotArgumentType
 	FIELD f_bsfrpwff UNKNOWN_SLOT_EXCEPTION Lcom/mojang/brigadier/exceptions/DynamicCommandExceptionType;
 	FIELD f_kfpmusct EXAMPLES Ljava/util/Collection;
+	FIELD f_opwacyge SINGLE_SLOT_ONLY_EXCEPTION Lcom/mojang/brigadier/exceptions/DynamicCommandExceptionType;
```
Comment (supersaiyansubtlety) [inline mappings/net/minecraft/command/argument/ItemSlotArgumentType.mapping]:
```suggestion
	FIELD f_opwacyge ONLY_SINGLE_ALLOWED Lcom/mojang/brigadier/exceptions/DynamicCommandExceptionType;
```
I don't think an `EXCEPTION` suffix is necessary here, but it does have a suffix it should be `EXCEPTION_TYPE`

**Review by supersaiyansubtlety — CHANGES_REQUESTED** [review]

**Comment by pyrox0** [comment]
All suggestions incorporated!

**Review by supersaiyansubtlety — APPROVED** [review]

File: mappings/net/minecraft/client/render/entity/model/WindChargeEntityModel.mapping [hunk 7]
```diff
@@ -0,0 +1,7 @@
+CLASS net/minecraft/unmapped/C_jnyjgoeq net/minecraft/client/render/entity/model/WindChargeEntityModel
+	FIELD f_dxnxrazm wind Lnet/minecraft/unmapped/C_rglqxnbw;
+	FIELD f_gzxypvoi bone Lnet/minecraft/unmapped/C_rglqxnbw;
+	FIELD f_pokzsptz wind_charge Lnet/minecraft/unmapped/C_rglqxnbw;
```
Comment (eristhea) [inline mappings/net/minecraft/client/render/entity/model/WindChargeEntityModel.mapping]:
```suggestion
	FIELD f_pokzsptz windCharge Lnet/minecraft/unmapped/C_rglqxnbw;
```

File: mappings/net/minecraft/network/codec/PacketEncoder.mapping [hunk 8]
```diff
@@ -1,4 +1,4 @@
 CLASS net/minecraft/unmapped/C_hiksfskl net/minecraft/network/codec/PacketEncoder
-	METHOD encode (Ljava/lang/Object;Ljava/lang/Object;)V
+	METHOD encode write (Ljava/lang/Object;Ljava/lang/Object;)V
```
Comment (eristhea) [inline mappings/net/minecraft/network/codec/PacketEncoder.mapping]:
```suggestion
	METHOD encode (Ljava/lang/Object;Ljava/lang/Object;)V
```
mapped by #557 as `encode` (you can't name it encode due to an enigma bug that was fixed in 2.2.1)

**Review by eristhea — CHANGES_REQUESTED** [review]

**Pull request merged**